lang/java/reef-runtime-multi/src/main/java/org/apache/reef/runtime/multi/client/MultiRuntimeDefinitionBuilder.java

```diff
@@ -33,15 +33,15 @@
  * Builder for multi runtime definition.
  */
 public final class MultiRuntimeDefinitionBuilder {
```

Comment:
Please mark the whole class as `@Unstable`. That allows us to follow up on our ideas for a better API when 0.16 comes around.